src/monitor/pg_auto_failover.c

```diff
@@ -77,10 +83,35 @@ _PG_init(void)
 						 "configuration variable in postgresql.conf.")));
 	}
 
+#if PG_VERSION_NUM >= PG_VERSION_15
```

Comment:
I guess you have not defined PG_VERSION_15. For Citus code, we have it here: https://github.com/citusdata/citus/blob/e2a94953342642f28d87e5ab846f55e29d636c50/src/include/distributed/pg_version_constants.h#L16